elm-package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.0.0",
+    "summary": "helpful summary of your project, less than 80 characters",
+    "repository": "https://github.com/user/project.git",
+    "license": "BSD3",
+    "source-directories": [
+        "."
```

Comment:
Forgot to mention that this is going to be problematic for importing modules within the project (even if set to `src`).  Since the stages are numbers we won't be able to import as `7.Menu` for example (leading with a number is syntactically incorrect).  We may need to put an `elm-package.json` in each stage.